src/protocols/secure_channel/StatusReport.h

```diff
@@ -95,6 +96,18 @@ class DLL_EXPORT StatusReport
     uint16_t GetProtocolCode() const { return mProtocolCode; }
     const System::PacketBufferHandle & GetProtocolData() const { return mProtocolData; }
 
+    /**
+     * Returns the minimum wait time if the status report is of type busy.
+     * If status report is not busy, then this is an empty optional.
+     * If parsing the minimum wait time fails, then this is an empty optional.
```

Comment:
If parsing the minimum wait time fails, `StatusReport::Parse` will fail, so people should not be examining its state anyway.....

Reply:
removed this line.